src/features/awkwardGuests/awkwardGuestsSlice.js

```diff
@@ -205,6 +217,28 @@ export const awkwardGuestsSlice = createSlice({
     updateLocations: (state, action) => {
       const { locationKey, key, value } = action.payload;
       state.locations[locationKey][key] = value;
+      if (state.locations[locationKey]["maybe"] && state.locations[locationKey]["notSeen"])
+      {
+        switch (key) {
+          case 'guests':
+            state.locations[locationKey]["maybe"] = state.locations[locationKey]["maybe"].filter(a => !state.locations[locationKey]["guests"].includes(a))
+            state.locations[locationKey]["notSeen"] = state.locations[locationKey]["notSeen"].filter(a => !state.locations[locationKey]["guests"].includes(a))
+            break;
+          case 'maybe':
+            state.locations[locationKey]["guests"] = state.locations[locationKey]["guests"].filter(a => !state.locations[locationKey]["maybe"].includes(a))
+            state.locations[locationKey]["notSeen"] = state.locations[locationKey]["notSeen"].filter(a => !state.locations[locationKey]["maybe"].includes(a))
+            break;
+          case 'notSeen':
+            state.locations[locationKey]["guests"] = state.locations[locationKey]["guests"].filter(a => !state.locations[locationKey]["notSeen"].includes(a))
+            state.locations[locationKey]["maybe"] = state.locations[locationKey]["maybe"].filter(a => !state.locations[locationKey]["notSeen"].includes(a))
+            break;
+          default:
+            break;
+        }
+        state.locations[locationKey]["maybe"].sort();
+        state.locations[locationKey]["notSeen"].sort();
+      }
+      state.locations[locationKey]["guests"].sort();
```

Comment:
this looks over-engineered.
- let's not keep track of two arrays each time. It is okay if the character is in guests and in maybe.
- instead of switch case, we can just do `state.locations[locationKey][key]` key can be "guests", "maybe", "notSeen"